client/package.json

```diff
@@ -16,7 +16,9 @@
     "bootstrap": "^4.1.3",
     "bootstrap-vue": "^2.0.0-beta",
     "cross-env": "^5.2.0",
-    "nuxt": "^2.0.0"
+    "npm-sass": "^2.3.0",
```

Comment:
Why did you have to add npm-sass and sass-loader? I don't think I needed those but please let me know why those we're required in your case.

Reply:
@fritsvt This is needed to fix this
https://github.com/fritsvt/laravel-nuxt-authentication/issues/6